exercises/run-length-encoding/.meta/template.j2

```diff
@@ -0,0 +1,36 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+{% macro test_case(case, tmod) -%}
+    {%- set input = case["input"] -%}
+    def test_{{ tmod }}{{ case["description"] | to_snake }}(self):
+        self.assertMultiLineEqual(
+            {% if case["property"] == "consistency" -%}
+            decode(encode("{{ ', '.join(input.values()) }}")),
+            {% else -%}
+            {{ case["property"] | to_snake }}("{{ ', '.join(input.values()) }}"),
+            {% endif -%}
+            "{{ case["expected"] }}"
+        )
+{%- endmacro %}
+
+{{ macros.header(['encode','decode'])}}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for supercase in cases -%}
+    
+    {%- set title_modifier = "" -%}
+    {%- if "encode" in supercase["description"] -%}
```

Comment:
Perhaps it would be more reliable to key off of the case property?

Example:
```Python
{% macro case_name(case) -%}
    test_
	{%- if case["property"] in ("encode", "decode") -%}
        {{ case["property"] }}_
    {%- endif -%}
    {{ case["description"} | to_snake }}
{%- endmacro %}

{% macro test_case(case) -%}
    {%- set input = case["input"] -%}
    def {{ case_name(case) }}(self):
# ...
```